assembly/activities/genReport.js

```diff
@@ -0,0 +1,39 @@
+import fs from "fs-extra";
+import path from "path";
+import { format } from "date-format-parse";
+
+export async function genReport(config) {
+  console.log(`generating docs assembly report...`);
+  let str = "";
+  str = `${str}# Docs Assembly Workflow report\n\n`;
+  str = `${str}Last assembled: ${format(new Date(), "dddd MMMM DD YYYY HH:mm:ss A ZZ")}\n\n`;
+  const sourceNodesPath = path.join(config.root_dir, config.temp_write_dir, config.source_info_nodes_file_name);
+  const sourceObjects = await fs.readJSON(sourceNodesPath);
+  const matchedGuidesPath = path.join(config.root_dir, config.temp_write_dir, config.attached_nodes_file_name);
+  const matchedGuides = await fs.readJSON(matchedGuidesPath);
+  let totalNodes = 0;
+  let totalGuides = 0;
+  for (const cfg of matchedGuides.cfgs) {
+    totalGuides++;
+    for (const section of cfg.sections) {
+      if (section.type == "langtabs") {
+        totalNodes = totalNodes + 4;
+      } else {
+        totalNodes++;
+      }
+    }
+  }
+  str = `${str}${totalGuides} guide configurations found.\n`;
+  str = `${str}${sourceObjects.length} information nodes found.\n`;
+  str = `${str}${totalNodes} information nodes are attached to guides.\n`;
+  str = `${str}Link magic has transformed the following node ids into site paths:\n`;
```

Comment:
```suggestion
  str = `${str}Link magic transformed the following node identifiers into site paths:\n`;
```
Should "magic" in "Link magic" be capitalized? (In other words, is Link Magic a thing? Or is link magic just something the code does?)